### DISCUSSION TITLE
How to configure Kong Manager GUI in the free plan not EE

### DISCUSSION THREAD

**Discussion by ngingihy** [Kong/kong#8356] [Help] [answered]
Hi, 
I've been following along the docs but it's not mentioned anywhere how to configure the GUI (Kong manager) in the **free** plan! How is this alleged to be most popular api-gateway when such a thing isn't well documented and not clear from the context if it's truly free.

**Comment by erSitzt** [comment]
Same here... i wanted to try out kong-gateway in one of my kubernetes clusters and from some of the videos i saw it should be possible to use some "basic features" in kong manager, but all i see is this... 
<img width="837" alt="image" src="https://user-images.githubusercontent.com/1625376/152507625-b1823d11-f84f-465c-9bec-f736ff5ab038.png">

So im not sure if this is normal, or if i should see at least a "default" workspace ? I cant create anything, there is not login and it does not say "Free" in the top banner as seen in other installations...

I'm using the Helm install, is there anything special to do there ?

**Comment by gAmUssA** [comment, marked as answer]
Hi @ngingihy 
I did an episode of Kong Builders last week wrt installing Kong using Helm.
I showed how enable kong manager with free version. Check it out https://youtu.be/lYm912ANi_E

Wrt vitals - vitals are enterprise feature (also requires database. I'm planning to walk through it next week. So, "subscribe and enable notifications"

**Reply by ilyesAj** [reply]
thank @gAmUssA for this session, but I'm pretty confused , in order to enable manger GUI you need to enable kong enterprise on the helm chart but you mention that it's part of free plan . I think the manager GUI section on helm chart should be decoupled from the entreprise section in order to avoid confusion
